testing/playwright/bootstrap-app.sh

```
@@ -0,0 +1,115 @@
+# 5. Kill any prior container of the same name.
+docker rm -f "$container" >/dev/null 2>&1 || true
+
+# 6. Launch the long-running Apache container.
+echo "bootstrap-app: starting container $container on https://127.0.0.1:$port"
+docker run -d --rm --name "$container" \
+    -v "$app_dir:/var/www/html" \
+    -p "127.0.0.1:$port:443" \
+    "$image" >/dev/null
+
+# 7. Poll for readiness. status.php returns {"status":"ok"} and does not require auth.
+for i in $(seq 1 30); do
```

Shellcheck (0.11.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)